grant.bib

```diff
@@ -16,6 +27,16 @@ @software{scikit-hep_uproot3_3_14_4
   url          = {https://zenodo.org/record/4537826}
 }
 
+@InProceedings{henry_schreiner-proc-scipy-2020,
```

Comment:
Citation taken from SciPy 2020 proceedings: https://doi.org/10.25080/Majora-342d178e-009

(feel free to harmonize the style, but I was lazy and left it as it was exported)